code/modules/vehicles/pussywagon.dm

```diff
@@ -54,45 +50,100 @@
 	if(href_list["empty_hoover"])
 		var/obj/vehicle/train/cargo/trolley/pussywagon/PT = tow
 		PT.empty_hoover(usr)
-	if(href_list["toggle_mops"])
+	if(href_list["toggle_mop"])
 		toggle_mop(usr)
 	SSvueui.check_uis_for_change(src)
 
-/obj/vehicle/train/cargo/engine/pussywagon/attackby(obj/item/W as obj, mob/user as mob)
+/obj/vehicle/train/cargo/engine/MouseDrop(user)
```

Comment:
I am not sure if its such a good idea to throw a "general" cargo/engine proc into the file for the janicart.

Reply:
typo. will fix.